21.md

```diff
@@ -0,0 +1,57 @@
+LUD-21: Pay in local unit of account.
+================================================
+
+`author: ethanrose`
+
+---
+
+The idea here is to enable a merchant to receive an exact payment amount, in whatever currency their goods are priced in (i.e. their own fiat currency). A sender should be able to input the amount in the recipient's own unit of account.
+
+Typically this is for a point-of-sale use-case where a lightning service is acting as a bitcoin-to-fiat payment processor. Currently, a sender would have to use their own calculator to determine how many sats to pay, and the recipient inevitably would receive the close-but-wrong amount. This bad experience is a hinderance to adoption of lightning as a cross-border payment system.
+
+## 1. `currency` record in payRequest details
+
+If `SERVICE` wishes for a `WALLET` user to specify a payment amount in a different unit of account, it MUST alter its JSON response in the payRequest details response to include a `currency` object, as follows:
+
+```diff
+ {
+   "callback": String,
+   "maxSendable": number,
+   "minSendable": number,
+   "metadata": string,
++  "currency": {
++    "code": "PHP",
++    "name": "Philippine Pesos",
++    "symbol": "₱",
++    "minSendable": 1,
++    "maxSendable": 50000,
++    "multiplier": 64501 // estimated millisats per "unit"
++  },
+   "tag": "payRequest",
+ }
+```
+
+NOTE: `SERVICE` must include a `multiplier` property. This is an estimate which helps the `WALLET` do frontend validation & set expectations for users. A simple helper for the `SERVICE` to calculate millisats `multiplier` from common BTC rates is:
+
+```
+10**11 / btcRate
+```
+
+## 2. User interface for specifying currency
+
+If there is a `currency` record in the initial response, `WALLET` must display a modified interface:
```

Comment:
```suggestion
If there is a `currency` record in the initial response, `WALLET` must display a modified interface:
```

You say *must* here, but since we do not omit `maxSendable` & `minSendable` from the initial payRequest details, it is actually a *should*.